api/v1alpha1/openapi.yaml

```diff
@@ -2569,6 +2569,42 @@ components:
               - filePath
           required:
           - httpRef
+    ApplicationSpec:
+      type: object
+      allOf:
+        - $ref: '#/components/schemas/ApplicationEnvVars'
+        - type: object
+          properties:
+            name:
```

Comment:
The current definition has `name` as optional. Shouldn't it be a required field?

Reply:
I agree the app should have a name (that's then shown in lists of apps in UI and CLI). Could we say that if the name is not specified, we default it to the image's base name?

Reply:
We have been back and forth I agree with Frank that in the image case we can just use the image basse name. But in gitops or inline is this as clear?